.ado/templates/e2e-testing-uwp.yml

```diff
@@ -0,0 +1,42 @@
+steps:
+  - task: DownloadSecureFile@1
+    name: UwpCertificate
+    inputs:
+      secureFile: FluentTester_TemporaryKey.pfx
+    displayName: "Download UWP App Certificate"
+
+  - powershell: |
+      certutil –f –p password –importpfx $(UwpCertificate.secureFilePath)
+    displayName: "Add Certificate to Personal Store"
+
+  - task: DownloadSecureFile@1
+    name: AzureCredentials
+    inputs:
+      secureFile: "installcredprovider.ps1"
+    displayName: "Download Azure Credentials Plugin for NuGet"
+
+  - task: PowerShell@2
+    inputs:
+      filePath: $(AzureCredentials.secureFilePath)
+      arguments: "-AddNetfx -Force"
+    displayName: "Install Azure Credentials Plugin for NuGet"
+
+  - script: |
+      yarn windows
```

Comment:
Might be good to add a comment here explaining why you need to go through this process of starting up the app and killing it, but leaving the server running.

Reply:
Updated comment to be more specific in newest commit.